runtime/bcutil/attrlookup.h

```diff
@@ -108,12 +86,6 @@ attributeHash (register const char *str, register unsigned int len)
   return len + asso_values[(unsigned char)str[1]];
 }
 
-#ifdef __GNUC__
```

Comment:
Is there a reason to remove these lines ?

Reply:
This is autogenerated from the changes attrlookup.gperf

Reply:
I think we want to keep these lines.